mix.exs

```diff
@@ -2,55 +2,68 @@ defmodule PropCheck.Mixfile do
   @moduledoc "Buildfile for PropCheck"
   use Mix.Project
 
+  @source_url "https://github.com/alfert/propcheck"
+  @version "1.3.1-dev"
```

Comment:
this requires also fixes in release.sh ...

Reply:
> this requires also fixes in release.sh ...

Done. Let me know if you understand your `release.sh` script correctly.

Reply:
Sorry for my late response, too much to do in real life. 

What I meant are the `sed` commands the replace the value of `version: ` but must now replace `@version `. If you feel uncomfortable with that change, don't mind. I can do it later.